### PR TITLE
Fix Readme

### PR DESCRIPTION
**Pull request by vidyasonawane**
## Changes proposed:
- wordsmithing
- grammatical corrections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Required keys can be obtained from [https://www.instagram.com/developer/authenti
 
 #### Google Maps
 
-Google maps is used to get information about location (info about country, city, latitude and longitude).
+Google maps are used to get information about location (info about country, city, latitude and longitude).
```
Comment (Kreijstal) [inline README.md]: In this cases the application "Google maps" is a singular application, there is only one of it.

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 It offers features for events with several tracks and venues. Event managers can create invitation forms for speakers and build schedules in a *drag and drop* interface. The event information is stored in a **database**. The system provides **API endpoints** to **fetch** the data, and to **modify** and **update** it. Organizers can import and export event data in a standard compressed file format that includes the event data in **JSON and binary** media files like **images and audio**.
 
-The **Open Event Server** exposes a well documented [JSON:API Spec](http://jsonapi.org/) Compliant `REST API` that can be used by external services *(like the Open Event App generators and the frontend)* to access & manipulate the data.
+The **Open Event Server** exposes a well documented [JSON: API Spec](http://jsonapi.org/) Compliant `REST API` that can be used by external services *(like the Open Event App generators and the frontend)* to access & manipulate the data.
```
Comment (Kreijstal) [inline README.md]: if you're gonna put a space in there, might as well remove the colon too.

Comment (poush) [reply README.md]: This seems right, no need to add space.

**Comment by CosmicCoder96** [comment]
Closing due to inactivity

**Pull request closed without merging**